Source/SwiftLintFramework/Rules/ControlStatementRule.swift

```diff
@@ -64,5 +69,28 @@ public struct ControlStatementRule: Rule {
                     "parentheses.")
                 }
         }
+
+    }
+
+    private func isFalsePositive(content: String, syntaxKind: SyntaxKind?) -> Bool {
+        if syntaxKind != .Keyword {
+            return true
+        }
+
+        let lastClosingParenthesePosition = content.lastIndexOf(")")
```

Comment:
This algorithm is pretty wasteful right now if `content.lastIndexOf(")")` is `-1`. If you change that function to return `nil` on not found, you could have `guard let lastClosingParenthesePosition = content.lastIndexOf(")") else { return false }` here.